man/git-reauthor.md

```diff
@@ -0,0 +1,83 @@
+git-reauthor(1) -- Rewrite history to change author's identity
+==============================================================
+
+## SYNOPSIS
+
+`git reauthor [<options>]`
+
+## DESCRIPTION
+
+Rewrite history to change author's identity.
+
+Goes through all existing commits and tags to selectively modify the identities:  
+- modify name and/or email  
+- affect author and/or committer identities  
+- specify the new name/email as parameters or use the ones from the Git config  
+- change all objects or only replace those from a specific identity (defined by its email)
+
+## OPTIONS
+
+  -A, --all
+
+    Rewrite all commits and tags.
+
+  -C, --config
+
+    Define correct values from user Git config
+    Parameters of --correct-email and --correct-name options take precedence
+    over the ones from the config if specified as well
+
+  -e, --correct-email &lt;<email>&gt;
+
+    Define the correct email to set
+    Empty email '' is allowed
+
+  -n, --correct-name &lt;<name>&gt;
+
+    Define the correct name to set
+    Empty name '' is not allowed
+
+  -o, --old-email &lt;<email>&gt;
+
+    Rewrite only commits and tags from old email
+    Empty email '' is allowed
+    If the --all flag is specified as well, it takes precedence over this option
```

Comment:
I think this can be slightly dangerous. Should we error out if both `--all` and `--old-email` are specified?
If someone specifies both it means they don't fully understand how the command works, and if it is the `--all` part they don't understand they have now rewritten more commits than they bargained for. If they then push to a remote, expecting the need of force pushing since they have rewritten some history, they might get into trouble.